studiopress/simple-social-icons
# Add TikTok social option

Add the option for a TikTok social icon.




### How to test

1. Open the Simple Social Icons plugin on this branch & open widget in dashboard
2. Add a link to the 'TikTok URI' form field
3. On the actual site, a TikTok icon should be present & when clicked, it should link to the correct place

### Documentation




### Suggested changelog entry

- Add TikTok social icon to default settings

### Notes
